### PR TITLE
add python link

### PR DESCRIPTION
**Pull request by xiangyan99**

### CODE REVIEW

File: docs/policies/README-TEMPLATE.md [hunk 1]
```diff
@@ -109,3 +109,8 @@ If you'd like to contribute to this library, please read the [contributing guide
 
 <!-- LINKS -->
 [style-guide-msft]: https://docs.microsoft.com/style-guide/capitalization
+
+
+## Links to language repos
+
+[Python Readme Template](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/template/azure-template/README.md)
```
Comment (jeremymeng) [inline docs/policies/README-TEMPLATE.md:116]:
```suggestion
[Python Readme Template](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/template/azure-template/README.md)
[JavaScript Readme Template](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/template/template/README.md)
```

File: docs/policies/README-TEMPLATE.md [hunk 2]
```diff
@@ -109,3 +109,9 @@ If you'd like to contribute to this library, please read the [contributing guide
...
+[JavaScript Readme Template](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/template/template/README.md)
```
Comment (alzimmermsft) [inline docs/policies/README-TEMPLATE.md:117]:
```suggestion
[Python Readme Template](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/template/azure-template/README.md)
[JavaScript Readme Template](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/template/template/README.md)
[Java Readme Template](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/template/azure-sdk-template/README.md)
```

File: docs/policies/README-TEMPLATE.md [hunk 3]
```diff
@@ -109,3 +109,10 @@ If you'd like to contribute to this library, please read the [contributing guide
...
+[Java Readme Template](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/template/azure-sdk-template/README.md)
```
Comment (weshaggard) [inline docs/policies/README-TEMPLATE.md:118]: If you are trying to have an example for each repo you should also add .NET.

**Comment by weshaggard** [comment]
What is the context for adding these? We already have a source link in the template up at the top.

File: docs/policies/README-TEMPLATE.md (hunk 3 above)
Comment (xiangyan99) [reply docs/policies/README-TEMPLATE.md:118]: @ShivangiReja

**Comment by xiangyan99** [comment]
> What is the context for adding these? We already have a source link in the template up at the top.

@weshaggard When we start a new SDK project, we want to have a template for README. Now we have two places for the README. One in the azure-sdk repo and the other one in the language repo. In order to void maintaining two copies, we add a link here. The one in language repo will be our main one.
Loop in @czubair for more information.

**Comment by weshaggard** [comment]
@xiangyan99 I would suggest the opposite direction as the template readme in azure-sdk is intended to contain all the common structure we want across all languages. So I'd suggest linking the readme to here and adding only a language specific example in the language repos.

**Comment by xiangyan99** [comment]
@weshaggard This is an option as well. And the purpose of this PR is to ask for your inputs. Thank you. :)

We will collect the inputs and @czubair will give the final guidance.

File: docs/policies/README-TEMPLATE.md (hunk 3 above)
Comment (ShivangiReja) [reply docs/policies/README-TEMPLATE.md:118]: Could you please add [.NET README template link](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/template/Azure.Template/README.md) here?

**Pull request merged**